ingest/validation/validate_metadata.py

```diff
@@ -180,43 +182,41 @@ def validate_cells_unique(metadata):
         valid = True
     else:
         dups = list_duplicates(metadata.cells)
-        msg = 'Error:  Duplicate CellID(s) in metadata file'
-        metadata.issues['error']['format'][msg] = dups
+        error_msg = 'Error:  Duplicate CellID(s) in metadata file'
```

Comment:
To prevent redundant text ("Error: Error: ") in issue messages:

```suggestion
        error_msg = 'Duplicate CellID(s) in metadata file'
```

Reply:
Thanks for the catch!

Reply:
Updated in d58de2c. Reference json updated for affected test.